MapboxCoreNavigation/RouteController.swift

```diff
@@ -390,9 +390,15 @@ extension RouteController: CLLocationManagerDelegate {
         }
         self.rawLocation = location
         
-        delegate?.routeController?(self, didUpdateLocations: [location])
+        if location.isQualified {
+            delegate?.routeController?(self, didUpdateLocations: [location])
+        }
         
         sessionState.pastLocations.push(location)
+        
+        guard location.isQualified else {
```

Comment:
We still want to collect the location, even though it's not qualified. However, we could move down L:393